etl/helpers.py

```diff
@@ -270,16 +278,44 @@ def create_dataset(
         if table.metadata.short_name in used_short_names:
             raise ValueError(f"Table short name `{table.metadata.short_name}` is already in use.")
         used_short_names.add(table.metadata.short_name)
+
+        from etl import grapher_helpers as gh
+
+        # Expand long to wide
+        if long_to_wide:
+            if ds.metadata.channel != "grapher":
+                log.warning("long_to_wide=True should be only used in grapher channel")
```

Comment:
I understand that we don't want to store big wide tables both in garden and grapher, but I don't think this should be a requirement. It's up to the user where the widening of the table happens. Or is there another reason why this should only happen in grapher?

Reply:
It's not a requirement, just a warning. I've added it because I mistakenly used it in the garden step and it took me a while to figure out the problem. I'll change the wording and if we want we can delete it later.